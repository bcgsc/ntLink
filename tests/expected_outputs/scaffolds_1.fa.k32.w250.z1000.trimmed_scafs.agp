ntLink_0	1	33803	1	W	188266	1	33803	+
ntLink_0	33804	38344	2	N	4541	scaffold	yes	paired-ends
ntLink_0	38345	87401	3	W	189231	1	49057	-
