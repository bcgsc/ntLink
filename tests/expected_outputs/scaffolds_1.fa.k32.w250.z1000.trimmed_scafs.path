ntLink_0	188266+ 4542N 189231-
