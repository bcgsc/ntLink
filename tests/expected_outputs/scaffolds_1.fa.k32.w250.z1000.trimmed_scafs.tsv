188266	0	33803
189231	0	49057
